modules/videopress/editor-media-view.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+/**
+ * WordPress Shortcode Editor View JS Code
+ */
+function videopress_handle_editor_view_js() {
+	global $content_width;
+	$current_screen = get_current_screen();
+	if ( ! isset( $current_screen->id ) || $current_screen->base !== 'post' ) {
+		return;
+	}
+
+	add_action( 'admin_print_footer_scripts', 'videopress_editor_view_js_templates' );
+
+	wp_enqueue_style( 'videopress-editor-ui', plugins_url( 'css/editor.css', __FILE__ ) );
+	wp_enqueue_script( 'videopress-editor-view', plugins_url( 'js/editor-view.js', __FILE__ ), array( 'wp-util', 'jquery' ), false, true );
+	wp_localize_script( 'videopress-editor-view', 'vpEditorView', array(
+		'home_url_host'     => parse_url( home_url(), PHP_URL_HOST ),
+		'min_content_width' => VIDEOPRESS_MIN_WIDTH,
+		'content_width'     => $content_width,
+		'modal_labels'      => array(
+			'title'     => esc_html__( 'VideoPress Shortcode', 'jetpack' ),
+			'guid'      => esc_html__( 'Video ID', 'jetpack' ),
+			'w'         => esc_html__( 'Video Width', 'jetpack' ),
+			'w_unit'    => esc_html__( 'pixels', 'jetpack' ),
+			/* Translators: example of usage of this is "Start Video After 10 seconds" */
+			'at'        => esc_html__( 'Start Video After', 'jetpack' ),
+			'at_unit'   => esc_html__( 'seconds', 'jetpack' ),
```

Comment:
Do you think we can combine these two strings and use something like `Start video after %s seconds` and just insert the input element using sprintf? This would give a lot more freedom to translators.